examples/grpc-routing/helloworld.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: grpc-infra-backend-v1
+spec:
+  selector:
+    app: grpc-infra-backend-v1
+  ports:
+  - protocol: TCP
+    port: 8080
+    targetPort: 50051
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: grpc-infra-backend-v1
+  labels:
+    app: grpc-infra-backend-v1
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: grpc-infra-backend-v1
+  template:
+    metadata:
+      labels:
+        app: grpc-infra-backend-v1
+    spec:
+      containers:
+      - name: grpc-infra-backend-v1
+        image: ghcr.io/nginxinc/kic-test-grpc-server:edge
```

Comment:
not that these containers change very much, but maybe we want to use a tag?

Reply:
Can we configure the tag to be maintained by dependabot, or will it require manual updating? At lease `edge` guarantees the latest version, I fear a tag could get out of date easily

Reply:
There's not a way to use dependabot for this...at least not a straightforward one.

I don't think anybody is going to change those containers (or publish new tags) anytime soon. At least with a tag we know that that specific one worked, with `edge` they could potentially break without us knowing about it.